README.md

```diff
@@ -1,85 +1,88 @@
-MicroPython Modules and Samples for XBee Devices
-================================================
+MicroPython Resources for XBee Devices
+======================================
 
-This project contains modules and sample code for use on XBee devices
-with MicroPython.  [Digi International][Digi] manages the project on
-GitHub as [xbee-micropython].
+This project contains modules and sample code for use on XBee devices with
+MicroPython. Digi has written some of the modules and samples, in addition to
+modifying existing code to address differences between the XBee and other
+MicroPython platforms. This project includes code from [micropython-lib].
```

Comment:
I don't remember 100%, but I think this link might not be valid Markdown syntax. I think you need either `[micropython-lib][]` or `[micropython-lib][micropython-lib]`.

Reply:
The MD plug-in was not complaining so I didn't realize about that. I'll change these kind of links. Thanks!

Reply:
Done.